pkgs/top-level/all-packages.nix

```diff
@@ -22660,7 +22660,10 @@ with pkgs;
 
   felix_remoteshell = callPackage ../servers/felix/remoteshell.nix { };
 
-  inherit (callPackages ../servers/firebird { }) firebird_4 firebird_3 firebird_2_5 firebird;
+  firebird_2_5 = callPackage ../servers/firebird/2_5.nix { };
+  firebird_3 = callPackage ../servers/firebird/3.nix { };
+  firebird_4 = callPackage ../servers/firebird/4.nix { };
+  firebird = firebird_4;
```

Comment:
Why do we need all 3 versions? Can we drop 2.5 at least?

Reply:
hydra build is already disabled for 2_5 and 3. legacy applications use it. I'd prefer to leave this as it is.